### PR TITLE
Fedora updates

### PR DESCRIPTION
**Pull request by kostyanf14**

### CODE REVIEW

File: .github/workflows/test-install.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        images: ['fedora:34', 'fedora:37', 'ubuntu:20', 'ubuntu:22', 'centos:9']
```
Comment (akihikodaki) [inline .github/workflows/test-install.yml:25]: You dropped old distros here but didn't for line 12.

File: autohck.sh [hunk 2]
```diff
@@ -116,31 +116,46 @@ get_fw_config() {
...
+      if [ ! -f "${OVMF_CODE_SB}" ]; then OVMF_CODE_SB='/usr/share/edk2/ovmf/OVMF_CODE_4M.qcow2'; fi
+
+      OVMF_VARS_SB='/usr/share/edk2/ovmf-4m/OVMF_VARS.secboot.fd'
+      if [ ! -f "${OVMF_VARS_SB}" ]; then OVMF_VARS_SB='/usr/share/edk2/ovmf/OVMF_VARS_4M.secboot.qcow2'; fi
+
+      # Fallback for Fedora 34
+      # Some OS can not boot with 2M OVMF image
+      if [ ! -f "${OVMF_CODE}" ]; then OVMF_CODE='/usr/share/edk2/ovmf/OVMF_CODE.fd'; fi
+      if [ ! -f "${OVMF_CODE_SB}" ]; then OVMF_CODE_SB='/usr/share/edk2/ovmf/OVMF_CODE.secboot.fd'; fi
+      if [ ! -f "${OVMF_VARS_SB}" ]; then OVMF_VARS_SB='/usr/share/edk2/ovmf/OVMF_VARS.secboot.fd'; fi
```
Comment (akihikodaki) [inline autohck.sh:144]: Do you really care Fedora 34?

**Review by akihikodaki — CHANGES_REQUESTED** [review]

File: autohck.sh (hunk 2 above)
Comment (kostyanf14) [reply autohck.sh:144]: Unfortunately, we have it on old servers where can't install new one

File: .github/workflows/test-install.yml (hunk 1 above)
Comment (kostyanf14) [reply .github/workflows/test-install.yml:25]: I dropped it only to check with the QEMU master branch. As we still use fedora:34 on the old server, I want to make sure that the installation will pass.

**Pull request merged**